src/main/java/org/rust/lang/core/psi/RustTokenElementTypes.java

```diff
@@ -203,6 +203,18 @@
         ANDAND
     );
 
+    @NotNull
+    TokenSet ALL_OPS = TokenSet.create(
```

Comment:
In IntelliJ's TokenSet language: `SPACE_AROUND_OPS = TokenSet.andNot(ALL_OPS, NO_SPACE_AROUND_OPS);`